meshes/v1/istio/spec.yaml

```diff
@@ -94,6 +94,18 @@ versions:
             - id: "strict-custom-cert"
               displayName: "Strict (Provided Cert)"
               description: "Enable mtls and provide a cert"
+              helmValues: "global:\n  controlPlaneSecurityEnabled: true\n  mtls:\n    enabled: true\ssecurity:\n selfSigned: false"
+              resourceDependencies:
```

Comment:
We could restrict refs in resource dependencies to the install namespace

Reply:
makes sense. would just expose the name in that case.